go.mod

```diff
@@ -21,8 +21,9 @@ require (
 	github.com/digitalocean/godo v1.61.0
 	github.com/gin-contrib/cors v0.0.0-20170318125340-cf4846e6a636
 	github.com/gin-contrib/static v0.0.0-20181225054800-cf5e10bbd933
```

Comment:
Can you update these components as well?

Reply:
updated